packages/main/src/themes/base/sizes-parameters.css

```diff
@@ -69,9 +69,6 @@
 	/* TextArea */
 	--_ui5_textarea_padding: 0.5625rem 0.6875rem;
 
-	/* Responsive Popover */
-	--_ui5-responnsive_popover_header_height: 2.75rem;
```

Comment:
Should we remove the cozy param? I think we should keep it and just fix the typo - responnsive -> responsive